compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -263,6 +263,28 @@ class CompilationTests {
 
       tests.foreach(_.delete())
     }
+
+    /* This tests for errors in the program's TASTy trees.
+    * The test consists of three files: (a) v1/A, (b) v1/B, and (c) v0/A. (a) and (b) are
+    * compatible, but (b) and (c) are not. If (b) and (c) are compiled together, there should be
+    * an error when reading the files' TASTy trees. */
+    locally {
+      val tastyErrorGroup = TestGroup("checkInit/tasty-error")
+      val tastyErrorOptions = options.without("-Xfatal-warnings")
+
+      val a0 = defaultOutputDir + tastyErrorGroup + "/A/v0/A"
+      val a1 = defaultOutputDir + tastyErrorGroup + "/A/v1/A"
+      val b1 = defaultOutputDir + tastyErrorGroup + "/B/v1/B"
```

Comment:
Minor: the names `a0`, `a1` and `b1` are not informative.

Reply:
Instead of assuming the output directory format, for maintainability, it would be better to explicitly set output directory based on the `defaultOutputDir`.

Fewer assumptions are better --- it causes fewer problems in case the format changes.